recipes-kernel/kernel-tests/kernel-test-fbomb-files/run-ptest

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./fbomb -d 24h
+
+exit 0
```

Comment:
NIT: I don't think it _strictly_ matters for the ptest runner, but it would probably be better to let the return code from `fbomb` propagate as the `run-ptest` return code. [See here](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/3560/Yocto-PTests?anchor=return-codes)